tictactoe.py

```diff
@@ -0,0 +1,113 @@
+#Initializing the board to a 3x3 dictionary
+board = {'1': ' ', '2': ' ', '3': ' ',
+         '4': ' ', '5': ' ', '6': ' ',
+         '7': ' ', '8': ' ', '9': ' '}
```

Comment:
Why did you choose to use a map instead of an array? An array already is a data structure that is indexed.  Also it would be nice to number the spaces so its easy for a user playing the game to understand when they enter a number what space their token will end up in.